components/org.wso2.carbon.identity.multi.attribute.login/org.wso2.carbon.identity.multi.attribute.login.resolver.regex/src/main/java/org/wso2/carbon/identity/multi/attribute/login/resolver/regex/RegexResolver.java

```diff
@@ -52,6 +53,9 @@ public ResolvedUserResult resolveUser(String loginAttribute, List<String> allowe
             UserRealm userRealm = UserResolverUtil.getUserRealm(tenantDomain);
             UniqueIDUserStoreManager userStoreManager = UserResolverUtil.getUserStoreManager(tenantDomain);
             ClaimManager claimManager = userRealm.getClaimManager();
+            if (allowedAttributes == null) {
+                return resolvedUserResult;
+            }
```

Comment:
shall we move it to the beginning of the try block?

Reply:
fixed with [26ef5c0](https://github.com/wso2-extensions/identity-governance/pull/580/commits/26ef5c032a3c5eb7a68ca846b5872b0563d0d793)